listenbrainz/webserver/static/js/jsx/follow-users.jsx

```diff
@@ -75,15 +75,15 @@ export class FollowUsers extends React.Component {
               <tr>
                 <th colSpan="2" width="50px">Order</th>
                 <th>User</th>
-                <th width="50%">Listening now</th>
-                <th width="25px"></th>
-                <th width="85px"></th>
+                <th>Listening now</th>
```

Comment:
Beat me to reporting it!

Reply:
I find this table quite ugly. Some redesign job in the future :)

Reply:
Yes, please!